objects/src/accounts/storage/mod.rs

```diff
@@ -64,7 +69,10 @@ impl AccountStorage {
     // CONSTRUCTOR
     // --------------------------------------------------------------------------------------------
     /// Returns a new instance of account storage initialized with the provided items.
-    pub fn new(items: Vec<SlotItem>) -> Result<AccountStorage, AccountError> {
+    pub fn new(
+        items: Vec<SlotItem>,
+        maps: Option<Vec<Smt>>,
```

Comment:
I think this would be better as a single `items`. Why did you split them into two arguments?

Reply:
I think items are something different. Items are the SlotItems. A SlotItem can be the root of a map.

Reply:
Originally, I was thinking that this would cary just the partial data - i.e., "simple values" and commitments to more complex values (i.e., storage maps and array). But now I think we can probably refactor this to carry full state of the storage.

One motivation for the original design was the idea that storage can get really big and we don't want to load all of it in memory when executing transactions. But I think there is a better way to handle vis something like #401.